buddies-server/lib/utils.js

```diff
@@ -0,0 +1,21 @@
+// ref: https://stackoverflow.com/a/48733909
+function compose(middleware) {
+  if (!middleware.length) {
+    return function(_req, _res, next) { next(); };
+  }
+
+  var head = middleware[0];
+  var tail = middleware.slice(1);
```

Comment:
This could be `const`

Reply:
I agree. I took the snippet without changing anything.

Reply:
Pushed a new commit to resolve this